core/src/test/resources/sql/planner.iq

```diff
@@ -20,6 +20,8 @@
 
 # The following line is used to update the default planner rules.
 # "+" means add a rule, "-" means remove a rule. Only the "CoreRules." can be omitted.
+# If multiple rules are involved, they need to be divided into multiple lines, one rule per line,
```

Comment:
I don't see a test using the new feature.

Reply:
I updated the documentation in `planner.iq`. This is a format specification to prevent subsequent developers from writing multiple formats.